### PR TITLE
Add buildpacks cli

### LINKED ISSUE #232: Add buildpacks CLI
## Expected Behaviour

Add buildpacks CLI [`pack`](https://github.com/buildpacks/pack) to arkade

## Current Behaviour

No buildpack CLI

## Possible Solution

Add `pack` to `tools.go`

### PR DESCRIPTION
**Pull request by yankeexe**
## Description

Added buildpacks CLI [`pack`](https://buildpacks.io/docs/tools/pack/)

Resolves #232 

## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?

Built a binary with the latest changes, tested `arkade` with : 

```bash 
$ ./arkade get pack
```

I tried installing `pack` and other CLI tools using `arkade` on Windows but got `uname` not found error. Do I need to do some pre-configuration?

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

File: pkg/get/tools.go [hunk 1]
```diff
@@ -415,5 +415,30 @@ https://releases.hashicorp.com/terraform/{{.Version}}/terraform_{{.Version}}_{{$
 https://github.com/{{.Owner}}/{{.Repo}}/releases/download/v{{.Version}}/gh_{{.Version}}_{{$osStr}}_{{$archStr}}.{{$extStr}}`,
 		})
 
+	tools = append(tools,
```
Comment (alexellis) [inline pkg/get/tools.go:418]: We will need a unit test for the app. Please can you see other examples like for faas-cli / inlets?

**Comment by alexellis** [comment]
Please unpick the cert-manager change whilst you are working on the above. I've already merged that via your other PR.

**Comment by yankeexe** [comment]
Hey @alexellis I have made the changes you asked for. Can you please check them and let me know if more changes are required 🙂

**Review by alexellis — APPROVED** [review]
LGTM

**Pull request merged**